Lib/slapdtest/_slapdtest.py

```diff
@@ -331,11 +352,14 @@ def _start_slapd(self):
         """
         Spawns/forks the slapd process
         """
+        urls = [self.ldap_uri]
+        if self.ldapi_uri:
+            urls.append(self.ldapi_uri)
         slapd_args = [
             self.PATH_SLAPD,
             '-f', self._slapd_conf,
             '-F', self.testrundir,
-            '-h', '%s' % ' '.join((self.ldap_uri, self.ldapi_uri)),
+            '-h', '%s' % ' '.join(urls),
```

Comment:
The `%` interpolation looks unnecessary here.

Reply:
Good catch